client-misk-jooq/src/main/java/com/squareup/backfila/client/base/jooq/ByteStringSerializer.java

```diff
@@ -0,0 +1,90 @@
+package com.squareup.backfila.client.base.jooq;
+
+import java.util.function.Function;
+import okio.ByteString;
+
+/**
+ * Responsible for serializing and deserializing values of a particular type into and from
```

Comment:
It might make sense to extract this at some point. This is fine for now.

Could you change the package name though. Maybe to something like `com.squareup.backfila.client.bytestring`
or `com.squareup.backfila.client.serialize`